### DISCUSSION TITLE
Any way to get *non* localized data from the REST API.

### DISCUSSION THREAD

**Discussion by richardvanbergen** [payloadcms/payload#139] [Q&A] [answered]
Hey guys,

I have a bunch of localized fields but there's one particular query where I need to get **all** available translations. The page `slug`. The reason being is I need to provide alternate language URLs and allow the user to switch between languages and each document can have multiple languages with independent URLs.

Currently the only way I can see to do that is loop over all available locales and query for each document of that locale to find out if it exists.

Is there any way for me to do that directly via the API? I couldn't see anything in the docs.

**Comment by jmikrut** [comment, marked as answer]
Hey @richardvanbergen —

We do have a way to do this!

`?locale=*`

Somehow that went un-documented. It only works via the REST API for now, because of GraphQL's strongly typed nature. Enabling this in GraphQL would mean that we have to make a whole other set of queries, which is possible, but likely lower on our priorities list.

We will get our docs updated ASAP. Give it a shot!

**Reply by richardvanbergen** [reply]
Thanks! It works quite nicely. 🙏
